activiti-cloud-starter-configure/src/main/java/org/activiti/spring/integration/RuntimeBundleActivityBehaviorFactory.java

```diff
@@ -0,0 +1,18 @@
+package org.activiti.spring.integration;
+
+import org.activiti.bpmn.model.ServiceTask;
+import org.activiti.engine.delegate.Expression;
+import org.activiti.engine.impl.bpmn.parser.factory.DefaultActivityBehaviorFactory;
+import org.activiti.engine.impl.delegate.ActivityBehavior;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RuntimeBundleActivityBehaviorFactory extends DefaultActivityBehaviorFactory {
```

Comment:
I do agree with @daisuke-yoshimoto this extension mechanism is not very flexibly. Users having their own factory behavior implementations will need to extend `RuntimeBundleActivityBehaviorFactory` instead of `DefaultActivityBehaviorFactory`. But I'm ok to merge it and come back to it later.

Reply:
@balsarori last question: is APS extending `DefaultActivityBehaviorFactory` or implementing a different `ActivityBehaviorFactory`

Reply:
I agree it's not very flexible but it's the simplest way to move ```${MQServiceTaskBehavior}``` for now, we can revise I more flexible approach later. 

In APS we extend ```ServiceTaskParseHandler``` and with changes in https://github.com/Activiti/Activiti/pull/1798 it will be easier to extend.

Reply:
Actually changes needed by APS were already merged in 5.x :)